apps/hpe-csi/data.yaml

```diff
@@ -0,0 +1,39 @@
+tags:
+  - Storage
+  - Drivers and plugins
+title: "HPE CSI Driver"
+summary: "TODO"
+logo: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
+description: |
```

Comment:
I see some charts have a `support_link`. Could we add one to https://scod.hpedev.io/legal/support/

Reply:
Sure, thank you